prime-router/metadata/fhir_mapping/hl7/datatypes/CWE/CodeableConcept.yml

```
@@ -0,0 +1,171 @@
+coding:
+  generateList: true
+  expressionType: nested
+  vars:
+    code: String, CWE.1
+    display: String, CWE.2
+    system: SYSTEM_URL, CWE.3
+    version: String, CWE.7
+  expressionsMap:
+    code:
+      condition: $code NOT_NULL
```

So this applies to all of them, but I think you can simplify and remove the condition if you write it like this?  Or at least all the tests still pass

```suggestion
      valueOf: CWE.1
      expressionType: HL7Spec
      type: String
```